crypto/pkcs7/pkcs7_test.cc

```diff
@@ -2144,6 +2145,7 @@ iWjrtmwM/HRbFEg2THS9b/vkiTsNSRCR9goaq9KPqXuJJsjJIoMA8IBHSLVvFnLf
       PEM_read_bio_PKCS7(bio.get(), nullptr, nullptr, nullptr));
   ASSERT_TRUE(pkcs7);
   ASSERT_TRUE(PKCS7_type_is_signed(pkcs7.get()));
+  ASSERT_TRUE(PKCS7_get0_signers(pkcs7.get(), nullptr, 0));
```

Comment:
here and above, we need to wrap the return value of this in a `bssl::UniquePtr`. I'm pretty sure this is what's causing [ASAN failures in the CI](https://us-west-2.codebuild.aws.amazon.com/project/eyJlbmNyeXB0ZWREYXRhIjoiWU01eG5zQlp3YllnS3pkTk94UmExM3lSMFBHeTE2dDdKNFVIVEoycUZuMXl1cnNraEs3dEFjTWpsWVdpY2g3L0FjUFpXMGNFaFNpUnpKbnBwOGpSaWVNTGhOZ3oxeGVYU0l4N0RSdHhPUGtscU1adiIsIml2UGFyYW1ldGVyU3BlYyI6IkxNS1lEZlRHT1NGNTRsSUciLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D/build/a454f689-3f91-4e10-bb9e-e51e509e09f5).